### PR TITLE
Add migration runner

### PR DESCRIPTION
**Pull request by abaniuszewicz**

### CODE REVIEW

File: backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                .ConfigureRunner(mrb =>
+                    mrb.AddPostgres()
+                        .WithGlobalConnectionString(sp => connectionString)
+                        .ScanIn(typeof(FluentMigratorInstaller).Assembly).For.Migrations())
+                .AddLogging(lb => lb.AddFluentMigratorConsole());
+        }
+
+        public static IHost MigrateUp(this IHost host)
+        {
+            var scope = host.Services.CreateScope();
```
Comment (Goush) [inline backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs:23]: IServiceScope implements IDisposable we might use using there

File: backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            var runner = scope.ServiceProvider.GetRequiredService<IMigrationRunner>();
+            runner.MigrateUp();
+            return host;
+        }
+
+        public static IHost MigrateUp(this IHost host, long version)
+        {
+            var runner = host.Services.GetRequiredService<IMigrationRunner>();
```
Comment (Goush) [inline backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs:31]: Add IServiceScope there so we can benefit from using IDisposable

File: backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            runner.MigrateUp(version);
+            return host;
+        }
+
+        public static IHost MigrateDown(this IHost host, long version)
+        {
+            var runner = host.Services.GetRequiredService<IMigrationRunner>();
```
Comment (Goush) [inline backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs:38]: Add IServiceScope there so we can benefit from using IDisposable

**Review by Goush — CHANGES_REQUESTED** [review]

File: backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs (hunk 1 above)
Comment (abaniuszewicz) [reply backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs]: Nice one, fixed

File: backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs (hunk 2 above)
Comment (abaniuszewicz) [reply backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs]: fixed

File: backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs (hunk 3 above)
Comment (abaniuszewicz) [reply backend/DevUp.Infrastructure.Postgres/Migrations/FluentMigratorInstaller.cs]: fixed

**Review by Goush — APPROVED** [review]

**Pull request merged**